### PR TITLE
Let there be only one info service

### PR DESCRIPTION
**Pull request by patflynn**
Here's what I did:

1. I took all the plugin info/configuration services from the cloud tools plugin and merged them into the account module equivalents.  We don't need two sets anymore since they're all in the same plugin.
1. I flattened all the hierarchical craziness that existed to support the two plugin services.

(oh and there's a google-java-format serialization format change that snuck in there. please forgive me.

### CODE REVIEW

**Comment by patflynn** [comment]
related to #1896

File: google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/startup/CloudToolsPluginInitializationComponentTest.java [hunk 1]
```diff
@@ -1,70 +0,0 @@
...
-import com.intellij.openapi.actionSystem.ActionManager;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.mockito.Mock;
-import org.mockito.runners.MockitoJUnitRunner;
-
-/** Tests to validate initialization on supported platforms */
-@RunWith(MockitoJUnitRunner.class)
-public class CloudToolsPluginInitializationComponentTest extends BasePluginTestCase {
```
Comment (etanshaul) [inline google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/startup/CloudToolsPluginInitializationComponentTest.java]: is there no longer an equivalent place for these tests?

**Comment by patflynn** [comment]
Thanks for noticing that!  That was a spurious delete I'm glad you caught.
Reinstated the test. pushing as soon as it passes locally.

**Comment by etanshaul** [comment]
strange about the merge conflict in CloudSdkAppEngineHelper. You must have started on a branch prior to integrating Ivan's changes.

**Comment by patflynn** [comment]
@etanshaul pushed both the test deletion revert and integrating Ivan's change.

**Pull request merged**